Data-Structure-ARRAYS.md

```diff
@@ -0,0 +1,62 @@
+#Arrays
+
+`array` is a kind of data structure that can store a fixed-size sequential collection of elements of the same type. An `array` is used to store 
+a collection of data, but it is often more useful to think of an `array` as a collection of variables of the same type.
+
+`array` consists of contiguous memory locations. The lowest address corresponds to the first element and the highest address to the last 
+element.
+
+## Arrays in Python
+
+Python doesn't have a native `array` data structure. An `array` in Python should not be confused with `list`. The major difference between a `list`
+and an `array` in Python is that a `list` can have different types of values whereas an `array` should have all the values of same type.
+
+####Declaration of `array`:
```

Comment:
~~####Declaration of `array`:~~ Add spaces between #### and Declaration of `array`: